packages/manager/test/SliceMachineManager-customTypes-updateCustomTypeMocksConfig.test.ts

```diff
@@ -0,0 +1,53 @@
+import { expect, it, vi } from "vitest";
+import { Buffer } from "node:buffer";
+
+import { createTestPlugin } from "./__testutils__/createTestPlugin";
+import { createTestProject } from "./__testutils__/createTestProject";
+
+import { createSliceMachineManager } from "../src";
+import { expectHookHandlerToHaveBeenCalledWithData } from "./__testutils__/expectHookHandlerToHaveBeenCalledWithData";
+
+it("saves a Custom Type's mock config as an asset", async () => {
+	const mocksConfig = { bar: "baz" };
+	const hookHandler = vi.fn();
+	const adapter = createTestPlugin({
+		setup: ({ hook }) => {
+			hook("custom-type:asset:update", hookHandler);
+		},
+	});
+	const cwd = await createTestProject({ adapter });
+	const manager = createSliceMachineManager({
+		nativePlugins: { [adapter.meta.name]: adapter },
+		cwd,
+	});
+
+	await manager.plugins.initPlugins();
+
+	const res = await manager.customTypes.updateCustomTypeMocksConfig({
+		customTypeID: "foo",
+		mocksConfig,
+	});
+
+	expect(res).toStrictEqual({
+		errors: [],
+	});
+	expectHookHandlerToHaveBeenCalledWithData(hookHandler, {
+		customTypeID: "foo",
+		asset: {
+			id: "mocks.config.json",
```

Comment:
it's mentioned in the code base a few times... so maybe

Reply:
Yeah, I noticed it throughout the codebase, but I don't think we actually have any Custom Type mocks. We re-implemented the feature in `@slicemachine/manager` just in case we needed to retain compatibility, but we could remove it if we don't need it.